op-e2e/interop/supersystem.go

```diff
@@ -469,9 +469,13 @@ func (s *interopE2ESystem) prepare(t *testing.T, w worldResourcePaths) {
 	s.l2s = s.prepareL2s()
 
 	// add the L2 RPCs to the supervisor now that the L2s are created
+	ctx := context.Background()
 	for _, l2 := range s.l2s {
-		err := s.SupervisorClient().AddL2RPC(context.Background(), l2.l2Geth.UserRPC().RPC())
-		require.NoError(s.t, err, "failed to add L2 RPC to supervisor")
+		// hack: wait for previous L2 additions to stop and start the supervisor again
+		// better would be to wait for the supervisor to be ready to accept new L2s
+		time.Sleep(10 * time.Second)
+		err := s.SupervisorClient().AddL2RPC(ctx, l2.l2Geth.UserRPC().RPC())
+		require.NoError(s.t, err, "failed to add L2 RPC to supervisor", "error", err)
```

Comment:
I had added this when I was seeing `context deadline timeout`, but I now realize that was due to extra things I had put in during debug, and this timing step wasn't needed.

`AddL2RPC` is fully synchronous and so this delay is meaningless and can/will be removed.

What I was witnessing was the `context deadline exceeded` causes the test setup to fail, but because the supervisor accepted the API request, it eventually does connect to the first L2 provided. I misinterpreted this to mean that the second call was failing because the backend wasn't up, but really it was failing on the *first* call.

All this is to say, this part turns out not to be needed and I'm removing it.